prometheus_client/exposition.py

```diff
@@ -105,6 +106,8 @@ def write_to_textfile(path, registry):
 def push_to_gateway(gateway, job, registry, grouping_key=None, timeout=None):
     '''Push metrics to the given pushgateway.
 
+    `gateway` is a url, but will assume http if no other scheme is provided.
```

Comment:
"scheme defaults to http if none is provided" would be a bit clearer.

If we're documenting one arg, they all should get documented. Exactly what bits of the URL are needed should be specified.